### ISSUE TITLE
Precision and Recall and mAP50 always zero when model is yolov8n.yaml (from scratch)

### ISSUE THREAD

**Issue by bzisl** [ultralytics/ultralytics#1242]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### YOLOv8 Component

Training

### Bug

I'm trying to train a custom dataset from zero (not using the pretrained weights), but always precision, recall, mAP is zero after 100 epochs. Using yolov8n.yaml (pertained), it works, but our previous results when we use scratch train always was better.

### Environment

Ultralytics YOLOv8.0.45 🚀 Python-3.8.10 torch-1.13.1+cu117 CUDA:0 (NVIDIA GeForce RTX 3090, 24268MiB)
Setup complete ✅ (32 CPUs, 125.7 GB RAM, 726.6/1758.9 GB disk)
OS: Ubuntu 22.04

### Minimal Reproducible Example

yolo/engine/trainer: task=detect, mode=train, model=yolov8/ultralytics/models/v8/yolov8n.yaml, data=xxxxx.yaml, epochs=600, patience=50, batch=70, imgsz=512, save=True, save_period=-1, cache=False, device=[0], workers=8, project=EVIL.AI/Motility/Boar, name=10xPHN_512x384x3n_r01, exist_ok=True, pretrained=False, optimizer=SGD, verbose=True, seed=19750129, deterministic=True, single_cls=False, image_weights=False, rect=True, cos_lr=False, close_mosaic=10, resume=False, min_memory=False, overlap_mask=True, mask_ratio=4, dropout=0.0, val=True, split=val, save_json=False, save_hybrid=False, conf=None, iou=0.7, max_det=3000, half=False, dnn=False, plots=True, source=None, show=False, save_txt=False, save_conf=False, save_crop=False, hide_labels=False, hide_conf=False, vid_stride=1, line_thickness=3, visualize=False, augment=False, agnostic_nms=False, classes=None, retina_masks=False, boxes=True, format=torchscript, keras=False, optimize=False, int8=False, dynamic=False, simplify=False, opset=None, workspace=4, nms=False, lr0=0.01, lrf=0.01, momentum=0.937, weight_decay=0.0005, warmup_epochs=3.0, warmup_momentum=0.8, warmup_bias_lr=0.1, box=7.5, cls=0.5, dfl=1.5, fl_gamma=0.0, label_smoothing=0.0, nbs=64, hsv_h=0, hsv_s=0, hsv_v=0.4, degrees=0.0, translate=0, scale=0.3, shear=0.0, perspective=0.0, flipud=0.0, fliplr=0.0, mosaic=0.0, mixup=0.0, copy_paste=0.0, cfg=None, v5loader=False, tracker=botsort.yaml, save_dir=...

### Are you willing to submit a PR?

- [X] Yes I'd like to help by submitting a PR!

**Comment by AyushExel** [comment]
Is this also happening with yolov5? We can only fix reproducible bugs so if its not happening with coco dataset then it might be related to your dataset

**Comment by bzisl** [comment]
It is not happening with YoloV5. In fact, I used YoloV5 always from scratch without any problem. With the same dataset used with YooV5, here always precision, recall, ... is zero. It is like the DNN parameters are always cero and they never change.Is there any scenario where this can happens? If usually people use the preweighted DNN, maybe we are not seeing a bug here.

**Comment by bzisl** [comment]
Ok. I'm going to try the CLI option. Currently I'm using it directly in Python and with ClearML support. I will check directly without ClearML and from CLI, and will investigate which is the difference. I will keep you informed.
Thanks for all!

**Comment by pderrenger** [comment]
@bzisl you're welcome! Feel free to reach out if you need further assistance. Good luck with your investigation! 😊
